app/src/main/java/com/example/android/receitaboa/activity/NovaReceitaFotoActivity.java

```diff
@@ -280,7 +277,9 @@ public void onDataChange(@NonNull DataSnapshot dataSnapshot) {
                     Intent i = new Intent(NovaReceitaFotoActivity.this, MainActivity.class);
                     startActivity(i);
 
-                    Toast.makeText(NovaReceitaFotoActivity.this,"Sua foto foi adicionada com sucesso!",Toast.LENGTH_SHORT).show();
+                    Toast.makeText(NovaReceitaFotoActivity.this,
+                            "Sua foto foi adicionada com sucesso!",
```

Comment:
Não se deve pôr uma string diretamente dentro do programa, extrair a string para um arquivo de tradução (string.xml)

https://www.andtuts.com/why-and-how-to-use-string-resources/